lib/public_key/src/pubkey_cert.erl

```diff
@@ -140,19 +140,50 @@ prepare_for_next_cert(OtpCert, ValidationState = #path_validation_state{
 %% current time.
 %%--------------------------------------------------------------------	  
 validate_time(OtpCert, UserState, VerifyFun) ->
+
+    % Parse and check validity of the certificate dates, and if it fails, invoke `verify_fun` to
+    % hand over control to the caller in order to decide what to do.
+    case parse_and_check_validity_dates(OtpCert) of
+        
```

Comment:
@adresanta I do not think we need these empty lines between the case clauses. No other cases in the application are written like that. Otherwise I think this is now looking good.